static/src/javascripts/projects/common/modules/commercial/dfp/breakout-iframe.js

```diff
@@ -22,57 +30,45 @@ define([
      */
     function breakoutIFrame(iFrame, $slot) {
         /*eslint-disable no-eval*/
-        var shouldRemoveIFrame = false,
-            $iFrame            = bonzo(iFrame),
-            iFrameBody         = iFrame.contentDocument.body,
-            $iFrameParent      = $iFrame.parent(),
-            type               = {};
-
-        if (iFrameBody) {
-            forEach(breakoutClasses, function (breakoutClass) {
-                $('.' + breakoutClass, iFrameBody).each(function (breakoutEl) {
-                    var creativeConfig,
-                        $breakoutEl     = bonzo(breakoutEl),
-                        breakoutContent = $breakoutEl.html();
-
-                    if (breakoutClass === 'breakout__script') {
-                        // new way of passing data from DFP
-                        if ($breakoutEl.attr('type') === 'application/json') {
-                            creativeConfig = JSON.parse(breakoutContent);
-                            if (creativeConfig.name === 'fluid250-v4' || creativeConfig.name === 'fluid250-v3') {
-                                creativeConfig.name = 'fluid250';
-                            }
-                            require(['common/modules/commercial/creatives/' + creativeConfig.name], function (Creative) {
-                                new Creative($slot, creativeConfig.params, creativeConfig.opts).create();
-                            });
-                        } else {
-                            // evil, but we own the returning js snippet
-                            eval(breakoutContent);
-                        }
+        var $iFrame            = bonzo(iFrame);
+        var $iFrameParent      = bonzo(iFrame.parentNode);
+        var iFrameBody         = iFrame.contentDocument.body;
+        var type               = {};
+        var $breakoutEl;
 
-                        type = creativeConfig.params.adType || '';
-
-                    } else {
-                        idleFastdom.write(function () {
-                            $iFrameParent.append(breakoutContent);
-                            $breakoutEl.remove();
-                        });
+        if (!iFrameBody) {
+            return type;
+        }
 
-                        $('.ad--responsive', $iFrameParent[0]).each(function (responsiveAd) {
-                            window.setTimeout(function () {
-                                idleFastdom.write(function () {
-                                    bonzo(responsiveAd).addClass('ad--responsive--open');
-                                });
-                            }, 50);
-                        });
-                    }
-                    shouldRemoveIFrame = true;
+        if (($breakoutEl = $('.breakout__html', iFrameBody)).length) {
+            fastdom.write(function () {
+                $iFrame.hide();
+                $breakoutEl.detach();
+                $iFrameParent.append($breakoutEl.html());
```

Comment:
Do we actually need to serialize the HTML and retokenize it? I'm pretty sure you can `.append` Bonzo collections directly.